am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/FilterResourceIntegrationTest.java

```diff
@@ -42,7 +43,7 @@ void setUp() {
 
     @Test
     void whenRowExistsAndHasReadPermissionsShouldReturnEnvelopeWithData() {
-        service.grantExplicitResourceAccess(createGrantForWholeDocument(resourceId, ACCESSOR_ID, READ_PERMISSION));
```

Comment:
I suggest reverting these changes. There is really no point changing all methods to accept set now. Let's take one element by default and lets overload method to accept list if needed. For most test cases it does not matter whether there is one or more accessors. Unnecessary changes IMO.

Reply:
👍

Reply:
Changes were made before overloaded :) can change back now.